.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: c
-script: asdf plugin-test golang .
+script: asdf plugin test golang . --asdf-plugin-gitref $TRAVIS_COMMIT go version
```

Comment:
This is an enhancement in testing script. This would ensure we are always testing against the latest commit in the current branch.

Reply:
I like it :+1: